vector/view.go

```diff
@@ -52,7 +52,7 @@ func NewView(index []uint32, val Any) Any {
 	return &View{val, index}
 }
 
-func nullsView(nulls *Bool, index []uint32) *Bool {
+func NullsView(index []uint32, nulls *Bool) *Bool {
```

Comment:
Nit: I've been meaning to change the NewView signature to `NewView(vec Any, index []uint32) Any` because that feels more natural so you might want to leave this as is.

Reply:
#5395 makes the NewView change.

Reply:
I just wanted to make this consistent with NewView so if you're going to change NewView I'll do as you say.